requirements.txt

```diff
@@ -1,7 +1,7 @@
 argopt
 construct
 cython
--e git+https://github.com/deepset-ai/haystack.git@cafa1230da57025746c1c9659c07dfc4b0008faf#egg=farm_haystack
```

Comment:
@AbdenourCh Can you check if the test pass with the newest haystack version ? I mean if you remove this hash an just do `pip install haystack` does it works ?

Reply:
I think it is good practice for us to freeze a Haystack version and not just use the latest one as it may introduce changes that we do not 'control'. I would install the latest release that is `v0.9.0`. Maybe @AbdenourCh, you could use this one ?

Reply:
ok, in that case maybe we can just do `haystack==0.9.0` instead of installing from git with git+https ?

Reply:
Using haystack==0.9.0 only does not work, because farm is not installed  generating this error
ModuleNotFoundError: No module named 'farm'

Using farm-haystack==0.9.0 it does not work, different version of transformers creating this problem 
ERROR: farm 0.8.0 has requirement transformers==4.6.1, but you'll have transformers 3.1.0 which is incompatible.

Reply:
As discussed, this error may originate from Farm and its compatibility with MAC or Windows. I changed the file `requirements.txt` so it includes `farm-haystack==0.9.0`. However the class names have changed on Hastack side's. As we discussed over the phone, you will change them.

Reply:
Has there been any progress on this? I rebased my local branches on master and I can't run piaf-ml anymore. How do you run piaf-ml at the moment?